packages/react/src/component-tests/IcTabs/IcTabsTestData.tsx

```diff
@@ -400,3 +400,35 @@ export const MinMaxTabs = (): ReactElement => {
     </div>
   );
 };
+
+export const DelayedTabs = (): ReactElement => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setData([
+        { label: "Ingredients" },
```

Comment:
For performance reasons declaring constants outside the function is good (in relation to caching), altough I see it breaks encapsulation. 

A good "nice-to-know" if you're in this situation other places outside the test suite.